rskj-core/src/main/java/co/rsk/peg/BridgeSerializationUtils.java

```diff
@@ -105,23 +102,27 @@ public static SortedMap<Keccak256, BtcTransaction> deserializeMap(byte[] data, N
         return map;
     }
 
-    public static byte[] serializeUTXOList(List<UTXO> list) throws IOException {
+    public static byte[] serializeUTXOList(List<UTXO> list) {
         int nutxos = list.size();
 
         byte[][] bytes = new byte[nutxos][];
         int n = 0;
 
-        for (UTXO utxo : list) {
-            try (ByteArrayOutputStream ostream = new ByteArrayOutputStream()) {
-                utxo.serializeToStream(ostream);
-                bytes[n++] = RLP.encodeElement(ostream.toByteArray());
+        try {
+            for (UTXO utxo : list) {
+                try (ByteArrayOutputStream ostream = new ByteArrayOutputStream()) {
+                    utxo.serializeToStream(ostream);
+                    bytes[n++] = RLP.encodeElement(ostream.toByteArray());
+                }
             }
+        } catch (IOException ioe) {
+            throw new SerializationException("Unable to serialize UTXO list ", ioe);
         }
```

Comment:
What is the reason behind using double `try` here? Moving the catch to the inner try seems to be enough. No need for the wrapper try.

Reply:
good point. Actually we should discuss what wilmer [pointed out](https://github.com/rsksmart/rskj/pull/2449#discussion_r1628045542). 
Do we want the method to serialize the utxos it can? Or should we stop the process if there's any we can't serialize?
I think the second approach is the correct one. If we want to de/serialize a list of utxos, we shouldn't be satisfied with the de/serialization of only _some_ of them

Reply:
you right, thanks natha